logr.go

```diff
@@ -314,6 +314,11 @@ func (l Logger) V(level int) Logger {
 	return l
 }
 
+// GetV returns the verbosity level of the logger.
+func (l Logger) GetV() int {
```

Comment:
Can this be a commit of its own, with tests to confirm logr.Discard().GetV() is 0 and .V(1) is 1 and .V(1).V(2) is 3 and .V(1).V(-1) is still 1.

Reply:
Yes. I had it separate initially, with a TODO in the first commit, but that then felt a bit silly. I can separate it again,

Reply:
Oh, you probably meant the logr.go change. Yes, that can be entirely separate.

Reply:
Done, and pushed.